package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "generator-ship-components-boilerplate",
+  "version": "0.1.0",
+  "description": "simple react boilerplate for the SHIP team",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ship-components/generator-ship-components-boilerplate/issues"
+  },
+  "homepage": "https://github.com/ship-components/generator-ship-components-boilerplate#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ship-components/generator-ship-components-boilerplate.git"
+  },
+  "scripts": {
+    "prepare": "gulp prepublish",
+    "test": "gulp",
+    "travis": "npm i trevor -g && trevor"
+  },
+  "author": {
+    "name": "Sepand Assadi",
+    "email": "sepand.assadi@sony.com"
+  },
+  "files": [
+    "generators"
+  ],
+  "main": "generators/index.js",
+  "keywords": [
+    "react boilerplate webapck redux router",
```

Comment:
Should this be individual values? also webapck needs to be fixed

Reply:
Fixed